share/chruby/chruby.sh

```diff
@@ -88,6 +88,7 @@ function chruby()
 				dir="${dir%%/}"; ruby="${dir##*/}"
 				case "$ruby" in
 					"$1")	match="$dir" && break ;;
+					"$1"*)	match="$dir" && break ;;
```

Comment:
Won't this pick the first `ruby-*` instead of the wanted last `ruby-*`, in case there are multiple like:
```
ruby-2.6.6
ruby-2.7.2
```

Could you add a test for that?

Reply:
You're right. It doesn't work and grabs `ruby-2.6.6` first.

However, the current logic of this case statement requires not matching and falling to the end.

So if `truffleruby` is installed and do `chruby ruby`, it will grab `truffleruby` because it is the last one ('t' comes after 'r'). This is for the current logic (without my pull request).

So, we would need to change the entire logic to parse the version number and grab the latest version number. This is outside of my capability in bash, so I don't know what to do.

Maybe I can just delete my pull request, and someone more knowledgeable in bash can make these changes. It will require a rewrite in the logic around this case statement parsing the version numbers.

Anyway, here is the new test code I used:

```bash
function test_chruby_grab_latest_version()
{
	RUBIES=("/path/to/ruby-2.6.6" "/path/to/ruby-2.7.2")

	local result=$(chruby "ruby" 2>&1)

	echo "$result"
}
```